src/q4wine-gui/iconsview.cpp

```diff
@@ -91,10 +91,7 @@ void IconsView::cmdOk_Click(){
 		sourceFile.append(lstIcons->currentItem()->text());
 
 		if (cbDefaultExport->checkState()==Qt::Checked){
-		saveFile.clear();
-		saveFile.append(corelib::getAppConfigLocation());
-		saveFile.append("/icons/");
-		saveFile.append(lstIcons->currentItem()->text());
+        saveFile = corelib::getAppConfigLocation(QStringList() << "icons" << lstIcons->currentItem()->text());
```

Comment:
This seems under-indented (same as other changes in this file), is it intentional?

Reply:
Good catch :)